packages/wix-storybook-utils/src/loader/prepareStory/prepareStory.spec.js

```diff
@@ -0,0 +1,320 @@
+  a: 1,
+  b: { ...stuff,
+    c: ['d']
+  },
+  d: {
+    e: {
+      f: {
+        hello: callMe('maybe')
+      }
+    }
+  },
+  _config: {
+    "i-am-config": "yes",
+    "nested": {
+      "oh": {
+        "boy": {
+          "thing": "hey there!"
+        }
+      }
+    },
+    storiesOf: storiesOf
+  }
+});`;
+
+      return expect(prepareStory(config)(source)).resolves.toEqual(expectation);
+    });
+
+    it('should work with module.exports', () => {
+      const source = 'module.exports = { a: 1 };';
+      const config = { a: 1 };
+      const expectation = `const story = require("wix-storybook-utils/Story").default;
+
+const {
+  storiesOf
+} = require("@storybook/react");
+
+module.exports = story({
+  a: 1,
+  _config: {
+    "a": 1,
+    storiesOf: storiesOf
+  }
+});`;
+
+      return expect(prepareStory(config)(source)).resolves.toEqual(expectation);
+    });
+
+    it('should work with referenced module.exports', () => {
+      const source = `
+        const stuff = { thing: { moreThings: ['hello'] } };
+        const callMe = () => true;
+        const reference = {
+          a: 1,
+          b: {
+            ...stuff,
+            c: ['d']
+          },
+          d: {
+            e: {
+              f: {
+                hello: callMe('maybe')
+              }
+            }
+          }
+        };
+
+        module.exports = reference;
+      `;
+
+      const config = {
+        'i-am-config': 'yes',
+        nested: {
+          oh: {
+            boy: {
+              thing: 'hey there!',
+            },
+          },
+        },
+      };
+
+      const expectation = `const story = require("wix-storybook-utils/Story").default;
+
+const {
+  storiesOf
+} = require("@storybook/react");
+
+const stuff = {
+  thing: {
+    moreThings: ['hello']
+  }
+};
+
+const callMe = () => true;
+
+const reference = {
+  a: 1,
+  b: { ...stuff,
+    c: ['d']
+  },
+  d: {
+    e: {
+      f: {
+        hello: callMe('maybe')
+      }
+    }
+  },
+  _config: {
+    "i-am-config": "yes",
+    "nested": {
+      "oh": {
+        "boy": {
+          "thing": "hey there!"
+        }
+      }
+    },
+    storiesOf: storiesOf
+  }
+};
+module.exports = story(reference);`;
+
+      return expect(prepareStory(config)(source)).resolves.toEqual(expectation);
+    });
+
+
+    it('should inject playgroundComponents require when given playgroundComponentsPath through storyConfig', () => {
+      const source = `
+      export default {};
+      `;
+
+      const config = {
+        playgroundComponentsPath: '/file'
+      };
+
+      const expectation = `import story from "wix-storybook-utils/Story";
+
+import { storiesOf } from "@storybook/react";
+
+export default story({
+  _config: {
+    "playgroundComponentsPath": "/file",
+    playgroundComponents: require('../../file').default,
+    storiesOf: storiesOf
+  }
+});`;
+
+      return expect(prepareStory(config, '/kazkas/kazkas')(source)).resolves.toEqual(expectation);
```

Comment:
maybe there's a way to have a more realistic test? it's not very clear how `/kazkas/kazkas` becomes `../../`, especially when `playgroundComponentsPath` points to `/file` - a file on root

also let's use english :)
```suggestion
      return expect(prepareStory(config, '/something/something')(source)).resolves.toEqual(expectation);
```

Reply:
oh no, I left lithuanian! its our secret language! how dare me